tests/run-make/dump-mono-stats/rmake.rs

```diff
@@ -0,0 +1,18 @@
+// A flag named dump-mono-stats was added to the compiler in 2022, which
+// collects stats on instantiation of items and their associated costs.
+// This test checks that the output stat file exists, and that it contains
+// a specific expected string.
+// See https://github.com/rust-lang/rust/pull/105481
+
+use run_make_support::{cwd, fs_wrapper, rustc};
+
+fn main() {
+    let path = cwd().display().to_string();
+    rustc()
+        .crate_type("lib")
+        .input("foo.rs")
+        .arg(format!("-Zdump-mono-stats={path}"))
```

Comment:
Suggestion: I think you could inline `path` here as

```rs
format!("-Zdump-mono-stats={}", cwd().display())
```

depending on what you consider more readable.